packages/sites/ortho-site/webapp/wdkCustomization/js/client/services.tsx

```diff
@@ -26,6 +31,12 @@ const orthoServiceWrappers = {
       method: 'get',
       path: `/group/${groupName}/layout`,
     }),
+  getGroupTree: (wdkService: WdkService) => (groupName: string) =>
+    wdkService.sendRequest(groupTreeResponseDecoder, {
+      useCache: true,
```

Comment:
It's worth noting that this property causes the response to be stored in indexeddb, which is governed by the browsers quota policy for domains. If the database exceeds the quota, attempts to cache responses will begin to fail (don't worry, this is handled for a seamless user experience). In all likelihood, important/critical responses will already be cached, so it should not impact website load times.

In the future, we might consider using LRU (least recently used) caches for things like this, where the cache container has an identifier, and we retain up to a certain number of entries, ejecting those used least recently.

For now, this is probably fine.

Reply:
Interesting. Are the cache entries keyed by the full request payload (like react-query)?

If so, this would seem to be a general WDK problem, not specific to fetching the tree (is it massively bigger than the responses from the other endpoints?)

Reply:
Entries are keyed by path and, if provided, cacheId. See [here](https://github.com/VEuPathDB/web-monorepo/blob/40eacc150830a81f32626d3e33893baa76c9fdd6/packages/libs/wdk-client/src/Service/ServiceBase.ts#L171). I don't know how big these newick strings can get, so it's hard to say how much of an issue it would be.

Again, I think this is probably fine, but I wanted to mention the current limitations.